web_src/js/features/formatting.js

```diff
@@ -11,4 +12,10 @@ export function initFormattingReplacements() {
       el.textContent = formatted;
     }
   }
+
+  // for each <time></time> tag, if it has the data-format="date" attribute, format
+  // the text according to the user's chosen locale
+  for (const timeElement of document.querySelectorAll('time[data-format="date"]')) {
+    timeElement.innerText = dateFormatter.format(new Date(timeElement.dateTime));
```

Comment:
```suggestion
    timeElement.textContent = dateFormatter.format(new Date(timeElement.dateTime));
```

`innerText` is kind of non-standard.